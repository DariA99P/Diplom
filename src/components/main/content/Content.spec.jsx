import React from 'react';
import { shallow } from 'enzyme';

import Content from '.';

describe('<Content />', () => {
  const tree = shallow(<Content />);

  it('should renders properly', () => {
    expect(tree).toMatchSnapshot();
  });
});
