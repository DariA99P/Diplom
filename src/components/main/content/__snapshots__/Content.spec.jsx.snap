// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should renders properly 1`] = `
<Adapter
  className="content"
/>
`;
